### ISSUE TITLE
HTTP request for outage start & end

### ISSUE THREAD

**Issue by olivernz** [louislam/uptime-kuma#3139]
### ⚠️ Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find similar feature request

### 🏷️ Feature Request Type

New Notification

### 🔖 Feature description

I have a Dynatrace UFO (https://www.dynatrace.com/solutions/devops/ufo/) which is basically a warning light LED for any issues that my systems have. The way it's driven is, that I can issue an API call to set the alert signal. I can imagine there are many solutions that work akin to that. But I can't control that directly in Uptime-kuma. I use a node.red setup to translate from webhook to HTTP calls. It would be nice to have this directly in Uptime-kuma without the excursion through node.red. 

### ✔️ Solution

Have a notification service that can do a HTTP request for outage start and a different one for outage end. 

### ❓ Alternatives

Currently going through Node.red. If anyone has a better idea I am all ears.

**Comment by abjoseph** [comment]
@olivernz, it isn't quite clear what the feature request is about from the description above. Are you asking for the addition of a separate webhook events, 1 for when an outage occurs and another for when the outage ends (for any given monitor of course)?

**Comment by olivernz** [comment]
Yes. That is the expectation. But a webhook doesn't quite work as it doesn't close the connection. I just need a normal HTTP(S) call.

**Comment by CommanderStorm** [comment]
> a webhook doesn't quite work as it doesn't close the connection. I just need a normal HTTP(S) call.
> 
> At the moment I have a separate script running that listens to the webhook and sends HTTP requests for heartbeat.status depending on what it is.

What do you mean exactly by not closing a connection?
Could you please share the script you wrote? 

Could this be a duplicate of #2413 (see #1991 too)?

**Comment by olivernz** [comment]
Could be some of the issue of #2413. Basically I can't make the webhook do a GET call. But even then I would need two GET calls. One for incident start and one for incident end. In that way it differs from #2413.

**Comment by CommanderStorm** [comment]
This is essentially a combination of these 3 issues:
- The ability to notifiy on incident creation/resolving: https://github.com/louislam/uptime-kuma/issues/1102
- The ability to filter what incidents are pushed to which monitor: https://github.com/louislam/uptime-kuma/issues/3585
- The ability to make GET-webhooks https://github.com/louislam/uptime-kuma/issues/2413

I think it would be better to close this issue in favour of the more specific issues
